src/test/regress/expected/multi_master_protocol_2.out

```diff
@@ -0,0 +1,34 @@
+--
+-- MULTI_MASTER_PROTOCOL
```

Comment:
Again, the diff between this and `multi_master_protocol.out` is extremely small:
```
18d17
<  CREATE INDEX lineitem_time_index ON public.lineitem USING btree (l_shipdate) TABLESPACE pg_default
19a19
>  CREATE INDEX lineitem_time_index ON public.lineitem USING btree (l_shipdate)
```

Why wouldn't the normalization rule fixed it?